flytekit/taskplugins/spark/task.py

```diff
@@ -13,18 +14,18 @@
 from flytekit.sdk.spark_types import SparkType
 
 
+@dataclass
 class Spark(object):
-    def __init__(self, spark_conf: Dict[str, str] = None, hadoop_conf: Dict[str, str] = None):
-        self._spark_conf = spark_conf
```

Comment:
ooh are we removing away from "_" member variable prefixes?

Reply:
?, I just converted this to a dataclass, basically dataclasses are automatically immutable